### ISSUE TITLE
resolv.conf not dynamically managed (12.04)

### ISSUE THREAD

**Issue by maxamg** [gds-operations/puppet-resolvconf#3]
I saw the following error when using this module:

```
goverac8:~# sudo /etc/init.d/networking restart
 * Running /etc/init.d/networking restart is deprecated because it may not enable again some interfaces
 * Reconfiguring network interfaces...                                                                                                         resolvconf: Error: /etc/resolv.conf isn't a symlink, not doing anything.
resolvconf: Error: /etc/resolv.conf isn't a symlink, not doing anything.
ssh stop/waiting
ssh start/running, process 16215
```

From reading [this](http://askubuntu.com/questions/137037/networkmanager-not-populating-resolv-conf) I decided to try running this:

```
sudo dpkg-reconfigure resolvconf
```

and explicitly selecting the option 'do not append original to dynamic file'.
This appears to reset `resolv.conf` as a link, then base, head, tail are populating `/etc/resolv.conf`. However the resulting file has additional commenting due to `/etc/resolvconf/resolv.conf.d/original` (which links to tail) also containing the comment:

```
deployer@goverac8:~$ cat /etc/resolv.conf
# Dynamic resolv.conf(5) file for glibc resolver(3) generated by resolvconf(8)
#     DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
nameserver 89.200.138.130
nameserver 78.31.111.10
nameserver 8.8.8.8
nameserver 8.8.4.4
# Dynamic resolv.conf(5) file for glibc resolver(3) generated by resolvconf(8)
#     DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
nameserver 10.0.2.3
search dmz.gds
```

The commenting issue appears to only happen in my LXC environment and the Memset VMs do not have any comments in the `original` file.

Alternatively we can:

```
sudo mv /etc/resolv.conf /run/resolvconf/resolv.conf
sudo ln -s ../run/resolvconf/resolv.conf /etc/resolv.conf
```

In summary:
1. Is `dpkg-reconfigure` necessary/best?
2. Problem with it appending `original` even when specifying no - im not sure where this toggle persists..
3. Any ideas on why a comment exists in `original` on an LXC container?
